Leetcode/Python/find pivot_index.py

```diff
@@ -0,0 +1,17 @@
+# Time Complexity : O(n)
```

Comment:
Please don't use spaces in your filename. 
Rename it please.

Reply:
yep